### ISSUE TITLE
Create flag that forbid to generate AI alt text

### ISSUE THREAD

**Issue by ischmittis** [marketing-factory/ai-filemetadata#5]
Some legal reasons might forbid to transfer images to external AI services. The reasons could be, that persons visible on the image have not accepted the use in AI context or Images contain intellectual properties. 

If a flag is set on image or folder level is set, the image won't be transfered to the ai service and no alt text is generated. If the flag is valid for a image the button in the gui should be disabled. 

This could be especially interesting for batch text generation as mentioned in https://github.com/marketing-factory/ai-filemetadata/issues/2

**Comment by ischmittis** [comment]
See #3 .
